### PR TITLE
Register s1d.is-a.dev

### PR DESCRIPTION
**Pull request by siiddhantt**
Register s1d.is-a.dev with A record pointing to 68.65.123.44.

### CODE REVIEW

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/s1d.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "siiddhantt",
+           "email": "siddhant.rai.5686@gmail.com",
+           "discord": "783600606451204176"
+        },
+    
+        "record": {
+            "A": ["68.65.123.44"]
```
[the remark this answers is not in the document]
Comment (siiddhantt) [reply domains/s1d.json:9]: siddhant-rai.web.app

this is the site, deployed using firebase

File: domains/s1d.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "siiddhantt",
+           "email": "siddhant.rai.5686@gmail.com",
+           "discord": "783600606451204176"
+        },
+    
+        "record": {
+            "CNAME": ["siddhant-rai.web.app"]
```
Comment (CuteDog5695) [inline domains/s1d.json:9]:
```suggestion
            "CNAME": "siddhant-rai.web.app"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems good to me now. (Also nice website!)

**Comment by siiddhantt** [comment]
> Seems good to me now. (Also nice website!)

Thanks for that :D

**Pull request merged**